.github/2.0/cookbooks/Hub.md

```diff
@@ -0,0 +1,155 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+Table of Contents
+
+- [Hubble CLI Guidelines](#hubble-cli-guidelines)
+  - [Prerequisites](#prerequisites)
+  - [1. Create Executor](#1-create-executor)
+  - [2. Push and Pull CLI](#2-push-and-pull-cli)
+    - [2.1 Distribute your executor](#21-distribute-your-executor)
+    - [2.2 Pull distributed executor](#22-pull-distributed-executor)
+  - [3. Use in Jina Flow](#3-use-in-jina-flow)
+    - [3.1 Using docker images](#31-using-docker-images)
+    - [3.2 Using source codes](#32-using-source-codes)
+    - [3.3 Override Default Parameters](#33-override-default-parameters)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# JinaHub Guidelines
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker) installed.
+- `jina[standard] >= 2.0.0` installed.
+    - Run `pip install -U "jina[standard]>=2.0.0"` to upgrade jina to correct version.
+
+## 1. Create Executor
+
+To create your Executor, you just need to run this command in your terminal:
+
+```terminal
+jina hub new
+```
+
+A wizard will ask you some questions about the Executor. For the basic configuration, you will be asked two things: The Executor’s name and where it should be saved. For this tutorial, we will call ours RequestLogger. And you can save it wherever you want to have your project. The wizard will ask if you want to have a more advanced configuration, but it is unnecessary for this tutorial.
+
+## 2. Push Executor to JinaHub
+
+If you want to share your executors to other persons or colleges, you can push your local executor to JinaHub.
+
+### 2.1 Basic
+
+```bash
+$ jina hub push [--public/--private] <path_to_executor_folder>
+```
+
+It will return an **UUID** as well as a **SECRET**. You would need these two items when using or update the executor later. **Please keep them carefully.**
+
+**Notes**:
+
+- When no `--public` or `--private` visibility options are provided. **By default, it's public.**
+- There is only one difference between `--public` and `--private`. You can use public executor once you know the `UUID`, but you must know `SECRET` and `UUID` if you want to use a private executor.
+
+### 2.2 Advanced
+
+```bash
+$ jina hub push <path_to_executor_folder> -t TAG1 -t TAG2 -f <path_to_dockerfile>
+```
+
+You can specify `-t` or `--tags` parameters to tag one executor. In additional, you can specify `-f` or `--docker-file` parameters to use a specific docker file to build your executor.
+
+## 3. Update Executor in JinaHub
+
+Everything is iterating in Internet world. We also provide a way to update your existing executor in JinaHub. To update one executor, you must have both `UUID` and `SECRET`.
+
+```bash
+$ jina hub push [--public/--private] --force <UUID> --secret <SECRET> <path_to_executor_folder>
```

Comment:
For better presentation, I think this should be explained before tags

Reply:
addressed